src/index.js

```diff
@@ -34,9 +35,9 @@ if (process.env.SENTRY_PUBLIC_DSN) {
 }
 
 if (
-  process.env.STEEMCONNECT_CLIENT_ID
-  && process.env.STEEMCONNECT_REDIRECT_URL
-  && process.env.STEEMCONNECT_HOST
+  process.env.STEEMCONNECT_CLIENT_ID &&
```

Comment:
I see prettier autoformatted this file :rofl:

Reply:
I'm not sure if it's a bad thing. We have to reformat files at some point.